### PR TITLE
Create

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: simple-semantic-search/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+![Vespa logo](https://vespa.ai/assets/vespa-logo-color.png)
+
+# Vespa sample applications - Simple semantic search
+
+A minimal semantic search application: 
+- Query and document text is converted to embeddings by the application. 
+- Search by embedding and/or text match.
+
+Requested at least Vespa 8.54.61.
```
Comment (lesters) [inline simple-semantic-search/README.md:11]: Required?

File: simple-semantic-search/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+</pre>
+
+Example queries:
+<pre>
+vespa query "yql=select * from doc where {targetHits: 100}nearestNeighbor(embedding, e)" "input.query(e)=embed(space contains many suns)"
+vespa query "yql=select * from doc where {targetHits: 100}nearestNeighbor(embedding, e)" "input.query(e)=embed(shipping stuff over the sea)"
+vespa query "yql=select * from doc where {targetHits: 100}nearestNeighbor(embedding, e)" "input.query(e)=embed(exchanging information by sound)"
+vespa query "yql=select * from doc where text contains 'boat'"
+vespa query "yql=select * from doc where {targetHits: 100}nearestNeighbor(embedding, e) AND text contains 'boat'" "input.query(e)=embed(exchanging information by sound)"
+</pre>
```
Comment (lesters) [inline simple-semantic-search/README.md:33]: Could suggest adding doc testing here, simple enough application for this

**Review by lesters — APPROVED** [review]
LGTM. Could consider adding doc testing later for this readme though.

**Comment by bratseth** [comment]
Yes, we should add testing. I want to get it out asap though

**Review by kkraune — COMMENTED** [review]
Great! I will add link checking to schema files in a separate PR.

**Comment by jobergum** [comment]
Thank you 🚀 🚀 🚀 👏 👏 👏 
image

**Pull request merged**